polaris-react/src/components/ResourceItem/ResourceItem.scss

```diff
@@ -120,38 +120,49 @@
     @include item-separator;
   }
 
-  &:not(.hasBulkActions):not(.selectMode) {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    &::after {
-      border-radius: var(--p-border-radius-05);
-    }
+  &.focused {
```

Comment:
Verified that removing the `hasBulkActions` and `.selectMode` selectors and using `.focused/.selectable` instead fixes the focus styles on ResourceList with BulkActions and introduces no regressions in the stories.